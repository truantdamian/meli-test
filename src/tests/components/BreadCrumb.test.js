import React from 'react';
import { shallow } from 'enzyme';
import BreadCrumb from './../../components/BreadCrumb/Breadcrumb'

describe('tests in Home', () => {
    test('should ', () => {
        const wrapper = shallow(<BreadCrumb categories={['comidas','ricasu']}/>);
        expect(wrapper).toMatchSnapshot();
    })
    
});
