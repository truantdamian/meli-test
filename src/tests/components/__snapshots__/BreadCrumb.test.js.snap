// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests in Home should  1`] = `
<div
  className="breadcrumb content"
>
  comidas|ricas
</div>
`;
